### DISCUSSION TITLE
Add libVLC or libmpv?

### DISCUSSION THREAD

**Discussion by mizzunet** [anilbeesetti/nextplayer#816] [Q&A]
Hi, I'm facing this issue https://github.com/anilbeesetti/nextplayer/issues/797. The issue is with ExoPlayer. Would you consider adding a fallback back end for such case? libVLC or libmpv?

Thanks

**Comment by blueberryapple** [comment]
What if instead users are able to select external media players similar to https://github.com/nova-video-player/aos-AVP? Thoughts @anilbeesetti?
